{"uuid": "d494bf83-de63-4934-a48e-668eb1774d92", "children": ["f7547698-3009-4182-a7fe-2f9da91e9751"], "befores": [{"name": "guest_token", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: Unsupported 'data' type: <class 'products.reqres.models.requests.token_request.GuestTokenRequest'>\n", "trace": "  File \"/home/ldl-koushic/.local/lib/python3.8/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ldl-koushic/.local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ldl-koushic/.local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/ldl-koushic/git/github/playwright_auto/conftest.py\", line 72, in guest_token\n    status_code, response = token_client.create_guest_token()\n  File \"/home/ldl-koushic/git/github/playwright_auto/products/reqres/clients/token_client.py\", line 26, in create_guest_token\n    return self.create_token(create_guest_token)\n  File \"/home/ldl-koushic/git/github/playwright_auto/products/reqres/clients/token_client.py\", line 18, in create_token\n    status_code, response = self.request_processor(endpoint)\n  File \"/home/ldl-koushic/git/github/playwright_auto/core/base/base_client.py\", line 49, in request_processor\n    response = self.request_context.post(url=url, headers=headers, data=request_body)\n  File \"/usr/local/lib/python3.8/dist-packages/playwright/sync_api/_generated.py\", line 18884, in post\n    self._sync(\n  File \"/usr/local/lib/python3.8/dist-packages/playwright/_impl/_sync_base.py\", line 109, in _sync\n    return task.result()\n  File \"/usr/local/lib/python3.8/dist-packages/playwright/_impl/_fetch.py\", line 248, in post\n    return await self.fetch(\n  File \"/usr/local/lib/python3.8/dist-packages/playwright/_impl/_fetch.py\", line 285, in fetch\n    return await self._inner_fetch(\n  File \"/usr/local/lib/python3.8/dist-packages/playwright/_impl/_fetch.py\", line 341, in _inner_fetch\n    raise Error(f\"Unsupported 'data' type: {type(data)}\")\n"}, "start": 1718706598438, "stop": 1718706598439}], "start": 1718706598438, "stop": 1718706598545}